spec.html

```diff
@@ -8498,6 +8536,12 @@ <h1>FunctionInitialize ( _F_, _kind_, _ParameterList_, _Body_, _Scope_ )</h1>
         1. Let _len_ be the ExpectedArgumentCount of _ParameterList_.
         1. Perform ! SetFunctionLength(_F_, _len_).
         1. If the source text matching _Body_ is strict mode code, let _Strict_ be *true*; else let _Strict_ be *false*.
+        1. Set _F_.[[AppearsInStackTraces]] to OmittedFromStackTraces of _Body_.
```

Comment:
Thanks. I was trying to avoid "negative" Boolean-valued slot names, but it seems to be more trouble than it's worth. Renamed to `OmittedFromStackTraces` for simplicity.